copy-webpack-plugin/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```

Comment:
whoops. done!

i copied the `tsconfig.json` from `html-webpack-plugin`, so maybe it should be fixed there, too.